src/__tests__/Footer.test.js

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from "@testing-library/react";
+import Footer from "../components/Footer";
+import { BrowserRouter } from "react-router-dom";
+
+describe("<Footer />", () => {
+  const footer = () => {
+    return render(
+      <BrowserRouter>
+        <Footer />
+      </BrowserRouter>
+    );
+  };
+  it("has a footer", () => {
+    footer();
+  });
+  it("has copyright information", () => {
+    footer();
+    const copyright = screen.getByRole("contentinfo");
+    expect(copyright).toBeInTheDocument();
+    const footerText = screen.getByText(/© cat tinder \| 2023/i);
```

Comment:
This test may fail in a month since your code is dynamic here and your test is static.  Something to keep in mind